src/OpenTelemetry.Collector.AspNet/AspNetCollector.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="AspNetCollector.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using OpenTelemetry.Collector.AspNet.Implementation;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Collector.AspNet
+{
+    /// <summary>
+    /// Requests collector.
+    /// </summary>
+    public class AspNetCollector : IDisposable
+    {
+        internal const string AspNetDiagnosticListenerName = "Microsoft.AspNet.TelemetryCorrelation";
+
+        private readonly DiagnosticSourceSubscriber diagnosticSourceSubscriber;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AspNetCollector"/> class.
+        /// </summary>
+        /// <param name="tracer">Tracer to record traced with.</param>
+        public AspNetCollector(Tracer tracer)
+            : this(tracer, new AspNetCollectorOptions())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AspNetCollector"/> class.
+        /// </summary>
+        /// <param name="tracer">Tracer to record traced with.</param>
+        /// <param name="options">Configuration options for dependencies collector.</param>
```

Comment:
typo - not dependencies.

Reply:
I fixed it here and on AspNetCore where I sourced it from. Will be on my next commit.